tests/wrappers/ReactWrapper-test.js

```diff
@@ -191,5 +192,41 @@ describe('ReactInVue', () => {
         )
       )
     })
+
+    it('works with no children', () => {
+      const spy = jest.spyOn(console, 'error');
```

Comment:
Uhh, semicolons!
Does `yarn lint` pass with this code? (Yes I have to fix Hound for PRs, sorry 😅)

Reply:
Lint does not catch the semicolons - sorry, was trying to remember to remove them, as there are no semicolons elsewhere. There are other lint errors (not introduced by this PR) however.